### DISCUSSION TITLE
Why is set not available in onRehydrateStorage?

### DISCUSSION THREAD

**Discussion by drewcoparker** [pmndrs/zustand#2854] [Q&A]
I cannot seem to reattach my actions when the store is rehydrated. Always get "set is not a function"

**Comment by dbritto-dev** [comment]
@drewcoparker would you mind sharing us a minimal repro on stackblitz?

**Comment by drewcoparker** [comment]
I'm using persist with vanilla createStore. Does the store exist when onRehydrateStorage is called? I'll try to provide a simplified repo when I get a chance.
